### PR TITLE
HAPP 434 Immunization detail screen UI

### PR DESCRIPTION
**Pull request by rashmi-bambhania**

### CODE REVIEW

File: app/src/main/res/drawable/ic_right_arrow.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:width="10dp"
-    android:height="11dp"
-    android:viewportWidth="10"
-    android:viewportHeight="11">
+    android:width="16dp"
```
Comment (amit-metri) [inline app/src/main/res/drawable/ic_right_arrow.xml:2]: This is being used home fragment with different color. This change will affect home fragment UI.

File: app/src/main/res/layout/fragment_immunization_record_detail.xml [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/space_2_x"
+                android:layout_marginEnd="@dimen/space_4_x"
+                android:layout_marginTop="@dimen/space_4_x"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toEndOf="@id/iv_vaccine"
+                app:layout_constraintTop_toTopOf="parent"
+                android:textSize="15sp"
+                android:textColor="@color/status_green"
+                android:text="Second dose of hepatitis A vaccine" />
```
Comment (amit-metri) [inline app/src/main/res/layout/fragment_immunization_record_detail.xml:61]: use string resource file for the text.

**Review by amit-metri — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/fragment_immunization_record_detail.xml (hunk 2 above)
Comment (rashmi-bambhania) [reply app/src/main/res/layout/fragment_immunization_record_detail.xml:61]: This is currently dummy text. Need to update after API and DB integration.

Comment (amit-metri) [reply app/src/main/res/layout/fragment_immunization_record_detail.xml:61]: Noted

**Review by amit-metri — APPROVED** [review]

**Pull request merged**